src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -326,10 +328,11 @@ class WorkflowManagerActor(backend: Backend) extends LoggingFSM[WorkflowManagerS
 
   private def workflowStdoutStderr(workflowId: WorkflowId): Future[Map[FullyQualifiedName, Seq[Seq[CallLogs]]]] = {
     def logMapFromStatusMap(descriptor: WorkflowDescriptor, statusMap: Map[ExecutionDatabaseKey, ExecutionStatus]): Try[Map[FullyQualifiedName, Seq[Seq[CallLogs]]]] = {
+      import cromwell.engine.finalcall.FinalCall._
       Try {
         val sortedMap = statusMap.toSeq.sortBy(_._1.index)
         val callsToPaths = for {
-          (key, status) <- sortedMap if hasLogs(statusMap.keys)(key)
+          (key, status) <- sortedMap if hasLogs(statusMap.keys)(key) && !key.fqn.isFinalCall
```

Comment:
Could maybe move the `isFinalCall` invocations up in the filtration order as they're super cheap